### DISCUSSION TITLE
Can I Load Themes Without the Screen Flickering?

### DISCUSSION THREAD

**Discussion by 1457384613gh** [dakanji/RefindPlus#56] [General]
**Describe the Bug:**

How to load theme smoothly rather than aglimmer and flickeringly?
The screen flashes three times before loading the theme. How can I prevent it from flickering?

**Comment by dakanji** [comment]
This is the effect of different things loading and cannot be avoided.
Use the RELEASE version to minimise this as it runs faster.

**Reply by 1457384613gh** [reply]
I know it takes time to load the graphical interface, too.

Like Clover, before graphface apearing, could you  set【black screen 3 seconds to wait】in place of【flash three times brightly】during loading the graphical interface of RefindPlus ？
This will become smooth, and look eyes-friendly and vision-preserving.

**Reply by dakanji** [reply]
Thanks for your feedback.
